### PR TITLE
Improve properties verification of ShadowAlgorithms #24749

### LINKED ISSUE #24749: Improve properties verification of ShadowAlgorithm
Hi community,

This issue is one of the tasks of #24746.

If not sure what to do, please refer to the example: #24742

## Aim
- [ ] ColumnRegexMatchedShadowAlgorithm
- [ ] ColumnValueMatchedShadowAlgorithm
- [ ] ~~SQLHintShadowAlgorithm~~

### PR DESCRIPTION
**Pull request by ZJJ087**
Fixes #24749.

Changes proposed in this pull request:
  -Improve properties verification of ShadowAlgorithm

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [x] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [x] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Comment by ZJJ087** [comment]
@Pace2Car pls review my code, thanks :)

**Comment by Pace2Car** [comment]
@ZJJ087  Hi, please keep the indentation of the blank line consistent with the previous line.

**Comment by ZJJ087** [comment]
> @ZJJ087 Hi, please keep the indentation of the blank line consistent with the previous line.

done

File: features/shadow/core/src/test/java/org/apache/shardingsphere/shadow/algorithm/shadow/column/ColumnRegexMatchShadowAlgorithmTest.java [hunk 1]
```diff
@@ -49,4 +50,17 @@ void assertExceptionCase() {
...
+    void assertPropertiesWithoutColumn() {
+        assertThrows(ShadowAlgorithmInitializationException.class, () ->
+                TypedSPILoader.getService(ShadowAlgorithm.class, "REGEX_MATCH", PropertiesBuilder.build(new Property("operation", "insert"), new Property("value", "[1]"))));
+    }
+
+    @Test
+    void assertPropertiesWithWrongOperation() {
+        assertThrows(ShadowAlgorithmInitializationException.class, () ->
+                TypedSPILoader.getService(ShadowAlgorithm.class, "REGEX_MATCH",
+                                          PropertiesBuilder.build(new Property("column", SHADOW_COLUMN), new Property("operation", "inser"), new Property("value", "[1]"))));
```
Comment (Pace2Car) [inline features/shadow/core/src/test/java/org/apache/shardingsphere/shadow/algorithm/shadow/column/ColumnRegexMatchShadowAlgorithmTest.java:64]: Replacing `inser` with `wrong` is more explicit.

File: features/shadow/core/src/test/java/org/apache/shardingsphere/shadow/algorithm/shadow/column/ColumnValueMatchShadowAlgorithmTest.java [hunk 2]
```diff
@@ -49,4 +50,17 @@ void assertExceptionCase() {
...
+    void assertPropertiesWithoutColumn() {
+        assertThrows(ShadowAlgorithmInitializationException.class, () ->
+                TypedSPILoader.getService(ShadowAlgorithm.class, "VALUE_MATCH", PropertiesBuilder.build(new Property("operation", "insert"), new Property("value", "1"))));
+    }
+
+    @Test
+    void assertPropertiesWithWrongOperation() {
+        assertThrows(ShadowAlgorithmInitializationException.class, () ->
+                TypedSPILoader.getService(ShadowAlgorithm.class, "VALUE_MATCH",
+                                          PropertiesBuilder.build(new Property("column", SHADOW_COLUMN), new Property("operation", "inser"), new Property("value", "1"))));
```
Comment (Pace2Car) [inline features/shadow/core/src/test/java/org/apache/shardingsphere/shadow/algorithm/shadow/column/ColumnValueMatchShadowAlgorithmTest.java:64]: as same

Comment (ZJJ087) [reply features/shadow/core/src/test/java/org/apache/shardingsphere/shadow/algorithm/shadow/column/ColumnValueMatchShadowAlgorithmTest.java]: done

**Review by Pace2Car — APPROVED** [review]

**Review by RaigorJiang — APPROVED** [review]

**Comment by RaigorJiang** [comment]
@Pace2Car @ZJJ087  Merged, thank you!

**Pull request merged**